### PR TITLE
[CI:DOCS] Fix small typo in windows docs

### PR DESCRIPTION
**Pull request by johanvdw**
Signed-off-by: Johan Van de Wauw <johan@gisky.be>

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by Luap99 — APPROVED** [review]
LGTM

File: docs/tutorials/podman-for-windows.md [hunk 1]
```diff
@@ -132,7 +132,7 @@ Port Forwarding
 
 Port forwarding also works as expected; ports will be bound against localhost
 (127.0.0.1). Note: When running as rootless (the default), you must use a port
-greater than 1023. See the Rooftull and Rootless section for more details.
+greater than 1023. See the Rootfull and Rootless section for more details.
```
Comment (rhatdan) [inline docs/tutorials/podman-for-windows.md:135]: Please spell Rootfull->Rootful

**Comment by mheon** [comment]
LGTM once comment from @rhatdan is addressed

**Comment by baude** [comment]
@johanvdw looks like if you make the change suggested, we merge your contribution as soon as possible

**Comment by rhatdan** [comment]
Now you need to squash your commits.
git rebase -i origin
git push --force
LGTM

**Comment by rhatdan** [comment]
You are missing a signature.

git commit -a --amend -s
git push --force

**Comment by rhatdan** [comment]
Thanks @johanvdw 
/lgtm

**Pull request merged**